src/components/FulfilmentPreference/FulfilmentPreference.js

```diff
@@ -0,0 +1,177 @@
+import PropTypes from 'prop-types';
+import { Field } from 'react-final-form';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Col,
+  Row,
+  Select,
+} from '@folio/stripes-components';
```

Comment:
stripes-components, stripes-smart-components, stripes-core imports should be imported like `@folio/stripes/components` etc

Reply:
Fixed